rpc/documentation/api.md

```diff
@@ -1476,3 +1477,88 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+## `TicketBuyerService`
+
+The `TicketBuyerService` service provides RPC clients with the ability to
+launch the automatic ticket purchaser with given initial configuration
+parameters, and to stop it.  It has no dependencies and is always running.
```

Comment:
instead of no dependencies, perhaps say that it is always running but depends on a loaded wallet